schemas/system/domains.schema

```diff
@@ -0,0 +1,22 @@
+{
+  IdStatus: Enum('Prealloc', 'Init', 'Actual', 'Historical'),
+  StorageKind: Enum('Master', 'Cache', 'Backup', 'Replica'),
+  CategoryRealm: Enum('Global', 'System', 'Local'),
+  CategoryFamily: Enum('Registry', 'Dictionary', 'Table', 'Log', 'View'),
+  UserAccess: Bitmask('Read', 'Insert', 'Update', 'Delete', 'Execute', 'Audit'),
+  Compression: Enum('None', 'GZIP', 'ZIP'),
+  SessionToken: { type: 'string', length: 64 },
+  MimeType: Enum(
```

Comment:
Do we really want to limit it like this? I understand that this has good intentions in mind but it may result in either this list growing out of hand or limiting the capacity of reporting/setting the needed mimetype. (Refs https://www.iana.org/assignments/media-types/media-types.xhtml)

Reply:
I think we will support only small subset of media types